promexporter/push.go

```diff
@@ -35,3 +35,16 @@ func PushMetrics(url string, job string) {
 		logger.Error("failed to push metrics: ", err)
 	}
 }
+
+// AddMetrics pushes all metrics in the default registry to the target URL
+// without overriding previously pushed metrics for this job
+//
+// The URL should contain no path for the official pushgateway
+func AddMetrics(url string, job string) {
+	client := &http.Client{}
+	client.Timeout = pushMetricsTimeout // default is no timeout
+	err := push.New(url, job).Gatherer(prometheus.DefaultGatherer).Client(client).Add()
+	if err != nil {
```

Comment:
I wonder if we should return error from function, rather than handling it here. I know it is done in similar fashion above, but it might be better to be able to provide an error to the function caller

Reply:
In that case I want to change function name in some code without additional actions.